caffe2/ideep/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ if(USE_MKL AND USE_IDEEP AND CAFFE2_COMPILER_SUPPORTS_AVX2_EXTENSIONS)
 
   add_library(Caffe2_ideep_operators OBJECT ${avx2_srcs})
   add_dependencies(Caffe2_ideep_operators Caffe_PROTO Caffe2_PROTO)
-  set_target_properties(Caffe2_ideep_operators PROPERTIES COMPILE_FLAGS "-mavx2")
+  set_target_properties(Caffe2_ideep_operators PROPERTIES COMPILE_FLAGS "-mavx2 -fopenmp")
```

Comment:
BTW, could you remind me why do we need `-mavx2` in IDEEP?

Reply:
yes. if have to impl optimized op required by caffe2 in ideep in time in future...

Reply:
Hmm, as far as I understand mkl-dnn will jit code based on the cpuinfo. Where do we need avx2 for ideep? Do you have an example?

Reply:
for example, channel_shuffle.
If mkldnn has no plan on it, avx2 intrinsic is an option to impl it in ideep for non-public format...

Reply:
I think `mkldnn` does have a plan for it.

Reply:
there should be a gap between a plan and code ready...
anyway, avx2 is optional. you can remove it for now.